src/components/player/ModeStatsGrid.vue

```diff
@@ -181,4 +172,14 @@ export default defineComponent({
 .tooltip-inner {
   white-space: pre-line;
 }
+
+.cell {
+  white-space: nowrap;
+
+  &.mode {
```

Comment:
Instead of creating this class, you can use vuetify html classes: `d-flex justify-center align-center`